app/services/analytics_events.rb

```diff
@@ -678,7 +678,7 @@ def idv_address_visit
   # @param [Boolean] acuant_sdk_upgrade_a_b_testing_enabled
   # @param [String] acuant_version
   # @param [Boolean] assessment
-  # @param [Integer] attempt number of attempts
+  # @param [Integer] captureAttempts number of attempts to capture / upload an image
```

Comment:
can we preserve the old name via comment?
```suggestion
  # @param [Integer] captureAttempts number of attempts to capture / upload an image (previously called "attempt")
```
that way if we ever want to run a query across old and new events, we have some hint of what it was

Reply:
That's a good idea, thank you!